package.json

```diff
@@ -1,11 +1,18 @@
 {
   "scripts": {
     "test": "xo && ava",
+    "build": "cd src && each '{*.js,!(libs)/**/*.js}' --parallel 10 -- rollup --config node:webext $PWD/%p --output $PWD/../extension/%p",
```

Comment:
I'm using `each` to create a copy of all the files in `src` to `extension` without having to manually configure each new file. The `!(libs)` is needed to skip the submodules we have, not just vendor files. 

Improvements to the glob are welcome though.